src/BloomBrowserUI/bookEdit/js/readerToolsModel.ts

```diff
@@ -123,6 +123,9 @@ class ReaderToolsModel {
       ReaderToolsModel.updateElementContent("stageNumber", "0");
       return;
     }
+    if (this.stageNumber > stages.length) {
+       this.stageNumber = stages.length;
+    }
```

Comment:
Is it possible for them to delete a stage other than the last? Seems if they delete the last stage, the others would not need adjusting, and if they can delete an earlier one, all the following ones would need adjusting, not just the one that is out of range.